internal/pkg/project/manifest/manifest_test.go

```diff
@@ -221,14 +225,14 @@ func fullJson() string {
   },
   "sortBy": "id",
   "naming": {
-    "branch": "{branch_id}-{branch_name}",
-    "config": "{component_type}/{component_id}/{config_id}-{config_name}",
-    "configRow": "rows/{config_row_id}-{config_row_name}",
-    "schedulerConfig": "schedules/{config_id}-{config_name}",
+    "branch": "{branch_name}",
+    "config": "{component_type}/{component_id}/{config_name}",
+    "configRow": "rows/{config_row_name}",
+    "schedulerConfig": "schedules/{config_name}",
```

Comment:
Upraveny test, vid `cases`, `naming: naming.TemplateWithoutIds(),` ... vacsia istota, ze sa neuklada iba default hodnota, ale to, co je realne zadane.